crates/notion-core/src/manifest/mod.rs

```diff
@@ -92,27 +91,33 @@ impl Manifest {
         toolchain: serial::ToolchainSpec,
         package_file: PathBuf,
     ) -> Fallible<()> {
-        // parse the entire package.json file into a Value
-        let file = File::open(&package_file).unknown()?;
-        let mut v: serde_json::Value = serde_json::from_reader(file).unknown()?;
+        let get_file = || package_file.to_string_lossy().to_string();
```

Comment:
A comment here explaining that we do it this way to allow easy repeated creation of the file path without moving it into the *other* closures below would be super helpful for future maintainers. It took me a minute of poking at it to figure out why we needed this.

Reply:
Great idea, updated!